Dockerfile

```diff
@@ -26,4 +26,6 @@ RUN adduser --uid 1000 --disabled-password --gecos '' reanauser && \
     chown -R reanauser:reanauser /code
 USER reanauser
 EXPOSE 5000
-CMD ["python", "reana_workflow_controller/app.py"]
+ENV FLASK_APP reana_workflow_controller/app.py
+ENV FLASK_DEBUG 1
```

Comment:
Do we want `FLASK_DEBUG` to be on by default? I think it's better to leave it out, and let the user set it on if wanted...